bin/strata-cli/src/net_type.rs

```diff
@@ -3,6 +3,7 @@ use std::str::FromStr;
 use console::Term;
 
 /// Represents a type of network, either Alpen's signet or Strata
+#[derive(PartialEq)]
```

Comment:
```suggestion
#[derive(PartialEq, Eq)]
```

I think that there is a `clippy` lint that if you are deriving `Partial[Eq|Ord]` it also says that you should probably also do `[Eq|Ord]`.